src/Microsoft.DotNet.Configurer/NuGetConfig.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.DotNet.Configurer
 {
     public class NuGetConfig : INuGetConfig
     {
-        public const string FallbackPackageFolders = "fallbackPackageFolders";
+        public const string FallbackPackageFolders = "packageSources";
```

Comment:
I filed an issue for https://github.com/dotnet/cli/issues/6507 and added a reference to this PR.